cmd/import.go

```diff
@@ -74,7 +79,66 @@ func evolveSchema(ctx context.Context, db string, coll string, docs []json.RawMe
 	return util.Error(err, "create or update collection")
 }
 
+func fixNumbers(ctx context.Context, coll string, docs []json.RawMessage) {
+	err := schema.Infer(&sch, coll, docs[0:1], PrimaryKey, AutoGenerate, 1)
+	util.Fatal(err, "infer schema")
+
+	if schema.HasArrayOfObjects {
```

Comment:
Weill all these error messages be visible to the user? All of this has to be hidden and transparent to the user. We should not expose any of this we are doing to the user.

Reply:
Those are debug messages only shown in the debug log.